posture/serialnumber_stub.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package posture
+
+import "errors"
+
+// GetSerialNumber returns client machine serial number(s).
+func GetSerialNumbers() ([]string, error) {
+	return []string{}, errors.New("not implemented")
```

Comment:
You probably don't need an empty slice and can `return nil, errors.New("not implemented")`?